src/semidiscretization/semidiscretization.jl

```diff
@@ -78,16 +78,18 @@ function calc_error_norms(u_ode, t, analyzer, semi::AbstractSemidiscretization,
 end
 
 """
-    semidiscretize(semi::AbstractSemidiscretization, tspan)
+    semidiscretize(semi::AbstractSemidiscretization, tspan; 
+                   reset_threads=true)
```

Comment:
Documenting `reset_threads` is unrelated to make the `RestartCallback` work for a `SemidiscretizationHyperbolicParabolic`, right?

Reply:
Yes - is there a good reason to not document that option? Felt a bit inconsistent for me.

Reply:
Not sure, but probably not?

Reply:
I think we should not document it: https://github.com/trixi-framework/Trixi.jl/pull/1584#issue-1817753586.